ocrd/ocrd/workspace_bagger.py

```diff
@@ -71,14 +72,11 @@ def _bag_mets_files(self, workspace, bagdir, ocrd_mets, processes):
                 if not file_grp_dir.is_dir():
```

Comment:
If I understand correctly, the `_bag_mets_files` method would still try to download every remote file if not locally available. How about providing a way to exclude some file groups here to disable automatic downloading for them? That way I would no longer have to remove problematic file groups (Goobi in my case) entries from the mets file.

Reply:
It may not be an efficient implementation but adding an extra argument `skip_filegroups` to `_bag_mets_files` which is a list of file groups and checking whether the current file `f` belongs to any of these would allow that. Then, the `bag` method should also accept that argument to allow control to the caller as well.

Reply:
Another option would be `download_filegroups` which is the opposite, and only downloads the specified file groups. If empty, should download all - the current behaviour.

Reply:
We did not originally allow incomplete OCRD-ZIP, which were supposed to be completely self-contained for long-term archiving purposes.

I started a general solution to that problem [years ago](https://github.com/OCR-D/core/compare/master...clone-filter) but never finished it. This would have included both whitelisting and blacklisting of search queries, i.e. a more general solution to what you propose, also for `ocrd workspace clone` and related functionality.

However, I see the need as well (these goobi/kitodo `file:/` URLs are a problem in our collection as well), so I am open to at least support fileGrp filtering for the bagger. I'll prepare a PR.